lib/OpenQA/Task/Needle/Save.pm

```diff
@@ -43,6 +43,8 @@ sub _json_validation {
     if (!exists $djson->{tags} || !exists $djson->{tags}[0]) {
         die 'no tag defined';
     }
+    my @not_ocr_area = grep { $_->{type} ne 'ocr' } @{$djson->{area}};
+    die 'Cannot create a needle only contains ocr type area' if scalar(@not_ocr_area) == 0;
```

Comment:
Better phrasing: "Cannot create a needle with only ocr areas"

Reply:
And capitalize OCR if you're amending this anyways.